src/scope.ml

```diff
@@ -164,6 +164,18 @@ let get_implicitness : p_term -> bool list = fun t ->
   in
   get_impl t
 
+(** [get_args t] decomposes the parser level term [t] into a spine
+   [(h,args)], when [h] is the term at the head of the application and [args]
+   is the list of all its arguments. WARNING: it does not decompose P_LLet,
+   P_UnaO, P_BinO and P_NLit. *)
```

Comment:
The indentation of the comment is bad, why did you not keep the one we had?
Also, the constructor names are not in brackets. I suggest using the following (which avoids listing constructors because that list may evolve):
```ocaml
(** [get_args t] decomposes the parser level term [t] into a spine [(h,args)],	
    when [h] is the term at the head of the application and [args] is the list	
    of all its arguments. Note that sugared applications (e.g., infix symbols)
    are not expanded, so [h] may still be unsugared to an application. *)
```

Reply:
To change the indentation please push directly on my branch.